contrib/stack/topsStack/Stack.py

```diff
@@ -260,20 +260,19 @@ def denseOffset(self, function):
         #self.f.write('ww : 256\n')
         #self.f.write('wh : 128\n')
 
-    def write_wrapper_config2run_file(self, configName, line_cnt, last_line = False, numProcess = 1):
+    def write_wrapper_config2run_file(self, configName, line_cnt = 1, last_line = False, numProcess = 1):
         # dispassionate list of commands for single process
         if numProcess == 1:
             self.runf.write(self.text_cmd + 'SentinelWrapper.py -c ' + configName + '\n')
         # aggregate background commands between wait blocks for speed gains
         elif numProcess > 1:
             self.runf.write(self.text_cmd + 'SentinelWrapper.py -c ' + configName + ' &\n')
-            if line_cnt == numProcess:
+            if line_cnt % numProcess == 0:
                 self.runf.write('wait\n\n')
                 line_cnt = 0
```

Comment:
This line is not needed anymore

Reply:
I think this must have been an earlier commit. Latest version looks like `if line_cnt % numProcess == 0 or last_line:`

Reply:
yes, you are right